src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,54 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: calendar-size($row-length, $gap-size, $padding);
+  padding: $padding;
+
+  &__day {
+    position: relative;
+    
+    @include square($cell-size);
+
+    margin: 0 $gap-size $gap-size 0;
+
+    background-color: $cell-color;
+    border: 1px solid black;
+
+    &:hover {
+      cursor: pointer;
+
+      animation: cell-hover 0.5s forwards;
+
+      @keyframes cell-hover {
+
+        100% {
+          background-color: $cell-color--active;
+          transform: translateY(-20px);
+        }
+      }
```

Comment:
It's a very simple animation, so it is better to use transition for it.
Also, after you remove hover, it should also work smoothly, not jump.

Reply:
nice catch. i did not notice that animation stopped abruptly after moving the cursor away.

fixed